.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Doc: https://github.com/redhat-plumbers-in-action/differential-shellcheck#readme
+
+name: Differential ShellCheck
+on:
+  pull_request:
+    branches: [main, v1.*, v2.*, v3.*, v4.*]
```

Comment:
I have noticed branches related to versions. Would you like to run a linter also against those branches?

Reply:
I don't think that would be a good idea.

Reply:
Ya, no need for that, it makes CI too complicated on the brain :grin:

Reply:
So let's run the linter only on `main`. Thanks for your feedback. :+1: